components/Footer.tsx

```diff
@@ -94,7 +105,7 @@ export const FooterImpl: React.FC = () => {
             target='_blank'
             rel='noopener noreferrer'
           >
-            <FaZhihu />
+            <GrDeploy />
```

Comment:
Not the line that I intended on changing, so this wasn't used at all and the deployment failed. I will just close it.